src/RuleTestCollection.php

```diff
@@ -32,16 +32,20 @@
 // Original Author of file: Walid Nouh
 // Purpose of file:
 // ----------------------------------------------------------------------
+
+namespace GlpiPlugin\Example;
+use RuleCollection;
+
 if (!defined('GLPI_ROOT')) {
    die("Sorry. You can't access directly to this file");
 }
 
 
-class PluginExampleRuleTestCollection extends RuleCollection {
+class RuleTestCollection extends RuleCollection {
 
    // From RuleCollection
    public $stop_on_first_match = true;
-   static public  $right       = 'rule_import';
+   public $right               = 'rule_import';
```

Comment:
CommonDBTM defines this property as not static

Reply:
var renamed as submitted [here](https://github.com/pluginsGLPI/example/pull/38#discussion_r926381095) for RuleTestCollection